drivers/i2s/i2s_mcux_sai.c

```diff
@@ -333,21 +332,19 @@ static void enable_mclk_direction(const struct device *dev, bool dir)
 	uint32_t offset = dev_cfg->mclk_pin_offset;
 	uint32_t mask = dev_cfg->mclk_pin_mask;
 	uint32_t value = 0;
+	uint32_t *gpr = (uint32_t *)DT_REG_ADDR(DT_NODELABEL(iomuxcgpr)) + offset;
 
 	if (iomuxgpr_dev == NULL) {
 		LOG_ERR("fail to find pinmux controller\n");
 		return;
 	}
```

Comment:
why is this required now?

Reply:
@gmarull if not defined it is NULL, and access this will have segmetation fault

Reply:
this file is no longer using a pinmux device, so please delete all related code.

Reply:
updated

Reply:
please, delete all pinmux related code: include, unused config fields